plugins/org.eclipse.emf.diffmerge.generic/src/org/eclipse/emf/diffmerge/generic/gdiffdata/impl/GComparisonImpl.java

```diff
@@ -1,976 +1,976 @@
-   * @see org.eclipse.emf.diffmerge.generic.api.IComparison#isConsistent()
-   * @generated NOT
-   */
-  public boolean isConsistent() {
-    for (Role role : Role.values()) {
-      if (!getDuplicateMatchIDs(role).isEmpty())
-        return false;
-    }
-    return true;
-  }
-
-  /**
-   * @see org.eclipse.emf.diffmerge.generic.api.IComparison#isThreeWay()
-   * @generated NOT
-   */
-  public boolean isThreeWay() {
-    return getAncestorScope() != null;
-  }
-
-  /**
-   * @see org.eclipse.emf.diffmerge.generic.api.IComparison#merge(org.eclipse.emf.diffmerge.generic.api.Role, boolean, org.eclipse.core.runtime.IProgressMonitor)
-   * @generated not
-   */
-  public Collection<IDifference<E>> merge(final Role destination_p,
-      boolean updateReferences_p, IProgressMonitor monitor_p) {
-    Collection<IDifference<E>> result = merge(new IMergeSelector<E>() {
-      /**
-       * @see org.eclipse.emf.diffmerge.generic.api.IMergeSelector#getMergeDirection(org.eclipse.emf.diffmerge.generic.api.diff.IDifference)
-       */
-      public Role getMergeDirection(IDifference<E> difference_p) {
-        return destination_p;
-      }
-    }, updateReferences_p, monitor_p);
-    return result;
-  }
-
-  /**
-   * @see org.eclipse.emf.diffmerge.generic.api.IComparison#merge(java.util.Collection, org.eclipse.emf.diffmerge.generic.api.Role, boolean, org.eclipse.core.runtime.IProgressMonitor)
-   * @generated NOT
-   */
-  public Collection<IDifference<E>> merge(
-      Collection<? extends IDifference<E>> differences_p, Role destination_p,
-      boolean updateReferences_p, IProgressMonitor monitor_p) {
-    MergeOperation<E> operation = new MergeOperation<E>(this, differences_p,
-        destination_p, updateReferences_p);
-    operation.run(monitor_p);
-    return operation.getOutput();
-  }
-
-  /**
-   * @see org.eclipse.emf.diffmerge.generic.api.IComparison#merge(org.eclipse.emf.diffmerge.generic.api.IMergeSelector, boolean, org.eclipse.core.runtime.IProgressMonitor)
-   * @generated NOT
-   */
-  public Collection<IDifference<E>> merge(IMergeSelector<E> merger_p,
-      boolean updateReferences_p, IProgressMonitor monitor_p) {
-    MergeOperation<E> operation = new MergeOperation<E>(this, merger_p,
-        updateReferences_p);
-    operation.run(monitor_p);
-    return operation.getOutput();
-  }
-
-  /**
-   * Create and return a mapping for this comparison
-   * @return a non-null mapping
-   * @generated NOT
-   */
-  protected abstract GMapping<E, A, R> newMapping();
-
-  /**
-   * @see org.eclipse.emf.diffmerge.generic.api.IComparison.Editable#swapScopes()
-   * @generated NOT
-   */
-  public boolean swapScopes() {
-    boolean isEmpty = getMapping().isEmpty();
-    if (isEmpty) {
-      IEditableTreeDataScope<E> formerTarget = targetScope;
-      targetScope = referenceScope;
-      referenceScope = formerTarget;
-    }
-    return isEmpty;
-  }
-
-} //GComparisonImpl
+/*********************************************************************

```

Comment:
Can you keep the original end of lines on GComparisonImpl, UnidirectionalStuff and EComparisonImpl 
(as it is a generated class)
You can convert back using the :
![image](https://user-images.githubusercontent.com/34767327/223658951-5d981bde-984b-4d07-a8d6-61a32dcc2539.png)